src/pages/case-studies/ibm-commerce-platform.mdx

```diff
@@ -0,0 +1,281 @@
+individual components, the page structure and layout was updated.
+
+With this approach, the Commerce team was able to refresh their product’s
+experience, set the stage for larger future changes, and complete the project in
+only three months.
+
+<Row>
+<Column colLg={12}>
+
+![Checkout screen in January 2020, April 2020, and the future design](images/iterate.png)
+
+<Caption>
+  From left to right: checkout screen in January 2020 before the transition,
+  April 2020 after the transition, and the future design.
+</Caption>
+
+</Column>
+</Row>
+
+## Benefits
+
+There have been several benefits from the transition to Carbon for both the
+Commerce team's own workflow and process, and the user experience. The team
+especially found value in the impact of their delivery speed and need for QA.
+
+### Faster delivery
+
+The entire Carbon refresh took only three months and since the transition, the
+team’s development cycle has sped up. By using Carbon, the developers are
+placing pre-built components and can therefore focus on more important issues
+instead of taking the time to build out each element and component.
+
+<Row>
+<Column colLg={12}>
+
+![Order confirmation designs and in production after one sprint](images/delivery.png)
+
+<Caption>
+  From left to right: the order confirmation screen handed off to developers,
+  the order confirmation screen in production at the end of one sprint, and the
+  order confirmation screen after in production at the end of the sprint plus
+  two extra working days.
+</Caption>
+
+</Column>
+</Row>
+
+### Minimal QA
+
+Once the team had been fully onboarded to Carbon, delivery required only minimal
+design QA because they were using components straight out of the box.
+
+### Continuity in experience
+
+The project vastly improved the continuity in user experience with other IBM
+touchpoints, including Support and My IBM. These experiences now feel continuous
+and intuitive for users.
+
+Within the Commerce Platform flow, pages are also unified and organized in the
+same way, providing familiarity and a feeling of dependability.
+
+<Row>
+<Column colLg={8}>
+
+![Continuity in experiences using Carbon for IBM.com.](images/continuity.png)
+
+<Caption>
+  Continuity in experiences using Carbon for IBM.com. From top to bottom:
+  Product configuration, Checkout, Confirmation, and Profile.
+</Caption>
+
+</Column>
+</Row>
+
+## Positive outcomes
+
+After the Carbon transition, more than 50% of customers are converting to
+purchase, which is on par with industry benchmarks. The conversion rate was
+raised by 5% which has led to a large increase in orders and revenue.
+
+## Next steps
+
+The Commerce Team is currently working on improvements experience of their
+configuration page. The next round of visual design improvements will focus on
+better aligning their interface with the Carbon for [IBM.com](http://ibm.com/)
+style models.
+
+## Related resources
+
+- [IBM.com](http://ibm.com/) makers, go to
```

Comment:
Do we want to use our resource tiles in here so the link to Carbon for IBM.com looks more official?